src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
There are lots of cases of this, where moving the comment implies different semantics for the comment. I seem to recall there's a problem with clang-format where it wants to indent the comment the same as the following #if -- is that true? In that case, maybe we could create a dummy statement like:

```
#define CLANG_FORMAT_BUG ;

...


     // this is a comment
     // more comment
     CLANG_FORMAT_BUG

#if ...
```


---
In reply to: [72540635](https://github.com/dotnet/coreclr/pull/6491#discussion_r72540635) [](ancestors = 72540635)